tests/conftest.py

```diff
@@ -2067,6 +2067,17 @@ def compare_running_config(pre_running_config, cur_running_config):
             return False
 
 
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
Is it necessary to add global autouse fixture? Probably put this fixture under tests/common. Import it to test scripts when it is needed?

Reply:
Agree, fixed in new commit